src/loggers.rs

```diff
@@ -76,14 +78,20 @@ where
     pub(crate) fn new(
         provider: &P,
         service_name: String,
+        enterprise_number: Option<String>,
         std_err_enabled: bool,
         host_name: String,
     ) -> Self {
+        let service_name_with_iana_number = match enterprise_number {
```

Comment:
I don't think `service_name_with_iana_number` is right name for this, since it implies it has an iana number but there's code paths where it might not.

I think here - and other places with the "iana_number" - we're better off with just calling it `service_name` and not making promises RE the iana or not, since it legit may not have them.

Reply:
I understand what you are saying,  but decided to stick with the naming as I need the unadorned `service_name` for OTEL log scope and the service_name, possibly with IANA enterprise_number, for the log line.